contracts/VToken.sol

```diff
@@ -463,8 +567,58 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         totalBorrows = totalBorrowsNew;
         totalReserves = totalReservesNew;
 
+        uint256 err = _accrueStableInterest(cashPrior, reservesPrior, blockDelta);
+
+        if (err != 0) {
+            return err;
```

Comment:
Should this throw instead and only return 0? since other services aren't consuming this yet do we need to return errors at all?

Reply:
Not needed here for now.